main/models/base_class.py

```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+
+from sqlalchemy import Column, DateTime, Integer
+from sqlalchemy.ext.declarative import declared_attr, declarative_base
+
+
+class Base(object):
+    @declared_attr
+    def __tablename__(cls):
+        return cls.__name__.lower()
+
+    id = Column(Integer, primary_key=True, index=True)
+    created = Column(DateTime, default=datetime.now)
+    updated = Column(DateTime)
```

Comment:
We should specify the default value for `updated` field every time we update the record